### DISCUSSION TITLE
"A newer version was found, but the install technology is different"

### DISCUSSION THREAD

**Discussion by AdamJel** [microsoft/winget-cli#1969] [General] [answered]
For some packages, I run into stdout:
```
A newer version was found, but the install technology is different from the current version installed. Please uninstall the package and install the newer version.
```

How can I get some more details into what is going on here. What exactly change? And perhaps, to choose to accept the change and to not be forced to reinstall the package completely?

**Comment by OfficialEsco** [comment, marked as answer]
This is the new prompt for `No applicable update found.` and is what i'm looking for
You can get the log from `winget --info`
> Logs: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir

This is most likely resolved by the `AppsAndFeaturesEntries` field OR we don't have the link to the other InstallerType.

I believe in the future you will get a option to install anyways

**Reply by eidylon** [reply]
This is SUCH a nice update! Love this message, it makes it very clear what happened. 
Used to `BUG. THE. CRAP!` out of me when it would show an update and then tell you no update found when you tried to install it. 

THANK YOU! 😎
